qbft/spectest/tests/create_message_spectest.go

```diff
@@ -17,8 +17,11 @@ const (
 )
 
 type CreateMsgSpecTest struct {
-	Name                                             string
-	Value                                            [32]byte
+	Name string
+	//TODO rename to root
+	Value [32]byte
+	//TODO rename to value
```

Comment:
The rename affects other test
after the PR is accepted I will open issues for the rename and remove the TODO comment